app/views/users/phones/edit.html.slim

```diff
@@ -6,16 +6,9 @@ h1.h3.my0 = t('headings.edit_info.phone')
     data: { unsupported_area_codes: unsupported_area_codes,
       international_phone_form: true },
     url: manage_phone_path) do |f|
-  = f.input :international_code,
-        collection: international_phone_codes,
-        include_blank: false,
-        input_html: { class: 'international-code' }
-  = f.label :phone
-    strong.left = t('account.index.phone')
-    span#otp_phone_label_info.ml1.italic
-      = t('devise.two_factor_authentication.otp_phone_label_info')
-  = f.input :phone, as: :tel, required: true, input_html: { class: 'phone', value: nil },
-    label: false
-  = render 'users/shared/otp_delivery_preference_selection'
-  = f.button :submit, t('forms.buttons.submit.confirm_change'), class: 'mt2'
+  = render 'users/shared/phone_input', f: f
+  = f.button :submit, t('forms.buttons.submit.confirm_change')
 = render 'shared/cancel', link: account_path
+
+= stylesheet_link_tag 'intl-tel-number/intlTelInput'
```

Comment:
do you want to include the js with the partial instead of in the calling view?

Reply:
For sure, thanks!